packages/neuron-ui/src/containers/Providers/index.tsx

```diff
@@ -121,7 +131,9 @@ const withProviders = (Comp: React.ComponentType<{ providerDispatch: ProviderDis
     <SettingsContext.Provider value={providers.settings}>
       <ChainContext.Provider value={providers.chain}>
         <WalletContext.Provider value={providers.wallet}>
-          <Comp {...props} providerDispatch={dispatch} />
+          <WalletsContext.Provider value={providers.wallets}>
```

Comment:
don't add a provider, just set wallets in settings.

Reply:
ok